note_module/output.py

```diff
@@ -0,0 +1,4 @@
+from execution import *
+
+
+user()
```

Comment:
Does this have a particular function. I see that you are not calling this anywhere in your other scripts.